synapse/storage/_base.py

```diff
@@ -650,20 +653,29 @@ def _simple_select_list_txn(cls, txn, table, keyvalues, retcols):
                 column names and values to select the rows with, or None to not
                 apply a WHERE clause.
             retcols (iterable[str]): the names of the columns to return
+            order (str): order the select by this column
+            order_asc (bool): order ascending (if order specified). Default True.
```

Comment:
TBH I would probably just have an `order` param that could be anything, e.g. `"foo ASC, bar DESC"`